features/reforger_upload.feature

```diff
@@ -14,6 +14,14 @@ Scenario: Upload next mission
   Then a new server config file is created
   And the config file is patched with <modlist.json> and <scenarioId>
 
+Scenario: Upload next mission and activate
+  Given a Zeusops mission locally ready
+  When Zeus calls "/zeus-upload"
+  And Zeus specifies <modlist.json>, <scenarioId>, <filename>, <activate>
```

Comment:
Potentially here we only care about the activate flag, as the rest is implied to be "used normally"?
In the Gherkin we don't have to be exhaustive, just make the meaning of the feature clear.

```suggestion
  When Zeus calls "/zeus-upload" with the activate flag
```